changelogs/fragments/67946-do_not_prefix_remote_path_if_windows

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
The filename needs to end with `.yml` or `.yaml` (that's what's causing CI to fail).